config/linux_gnu.py

```diff
@@ -8,14 +8,15 @@
 #flags_arch = '-g -fprofile-arcs -ftest-coverage' # gcov
 flags_arch = '-O3 -g -ffast-math -funroll-loops -fPIC -pedantic'
 #flags_arch = '-Wall -O3 -g'
-#flags_arch = '-Wall -O0 -g'
+flags_arch = '-Wall -O0 -g'
```

Comment:
The reason that the VL+CT tests time out is that this line became un-commented. The particular test that timed out executes 9 simulations. Turning off all optimizations really made all of the simulation time add up.

I sshed into circleci to test this. The test runs in just under 2-minutes if you comment this line again.

Reply:
As a minor followup, it's not currently my intention to review this PR. I'm not really familiar with the particle machinery or any of the Enzo functionallity that's being ported. 

So I'm not sure how much useful my review would actually be.

Reply:
Thanks for this catch!